libraries/core/src/main/java/com/paritytrading/philadelphia/FIXValue.java

```diff
@@ -346,6 +347,18 @@ public String asString() {
         return new String(bytes, offset, length, US_ASCII);
     }
 
+    /**
+     * Get the value as a string. The value is appended to the provided
+     * appendable.
+     *
+     * @param x an appendable
+     * @throws IOException if an I/O error occurs
+     */
+    public void asString(Appendable x) throws IOException {
```

Comment:
Would it make sense to remove the method `asString(StringBuilder x)` now, or not due to backwards compatibility?

Reply:
Removing that method would introduce a binary incompatible change. Also, it's worth noting that `Appendable#append` may throw an `IOException` whereas `StringBuilder#append` may not.